### DISCUSSION TITLE
World from database

### DISCUSSION THREAD

**Discussion by DevBlokJochem** [Minestom/Minestom#1325] [Help] [answered]
Hello, is there a way to get/store the world data from/to a database?

This is my situation: I have 3 game servers who all needs the same maps. I want to edit this map in a bukkit server (maybe in a minestom server for quick changes), export it to the database, so it could be loaded into my instance (Maybe it is smarter to save the world data first in a new folder).

**Comment by MrlnHi** [comment, marked as answer]
Create your own format that fits in a database or use TNT by emortal

**Reply by dev-hydrogen** [reply]
Isnt TNT abandoned?

**Reply by DevBlokJochem** [reply]
No, he re-opened that.

**Comment by OcZi** [comment]
There is a world format called Slime by the Slime World manager that supports databases. (not designed for survival worlds)
But obviously someone needs to implement it or yourself (refactor all bukkit code to minestom)
